lib/vssolution.cpp

```diff
@@ -1354,16 +1400,14 @@ void VisualizationSceneSolution::Prepare()
          PrepareFlat2();
          return;
       default:
-         if (v_normals)
+         if (v_normals && dim > 1)
          {
             PrepareWithNormals();
```

Comment:
Regarding this change here I am not super confident. This change will need intervention further intervention for the line rendering PR to setup the normals of the flat representation correctly.